doc/man7/EVP_RAND-TEST-RAND.pod

```diff
@@ -0,0 +1,109 @@
+=pod
+
+=head1 NAME
+
+EVP_RAND-TEST-RAND - The test EVP_RAND implementation
+
+=head1 DESCRIPTION
+
+Support for a test generator through the B<EVP_RAND> API. This generator is
```

Comment:
Side note: We should consider configuring this for all our fuzz tests.

Reply:
And our unit tests.